app/controllers/defence_requests_controller.rb

```diff
@@ -1,6 +1,7 @@
 class DefenceRequestsController < BaseController
 
   include DefenceRequestConcern
+  include DefenceRequestsHelper
```

Comment:
Yeah, so this is something I'm not sure about. Essentially `ApplicationHelper` is automatically available everywhere and I thought we're just splitting it semantically to smaller focused helpers. But if the purpose of all other helpers apart from `ApplicationHelper` is just views, than I'll move my url helper there.